src/types.ts

```diff
@@ -240,12 +240,7 @@ export interface AclPermission {
  * App Installer types
  */
 
-export type NetworkType =
-  | 'homestead'
-  | 'rinkeby'
-  | 'ropsten'
-  | 'kovan'
-  | 'goerli'
+export type NetworkType = 'mainnet' | 'rinkeby' | 'ropsten' | 'kovan' | 'goerli'
```

Comment:
I also think this is likely a breaking change and we should at least fall back to 'homestead' being equivalent to 'mainnet'.

Reply:
AFAIK this is the only place this `NetworkType` is used and it works when specifying mainnet as 'mainnet' and not 'homestead'